research-software-directory/RSD-as-a-service
# fix: reserve add slug as used

# Used cannot create software or project using "add" slug

Closes #1338 

Changes proposed in this pull request:
* Add software and add project validation includes check for "add" slug as this slug is reserved   
* The slug validation feedback to user is improved. From the moment the user start typing/changing slug until the end of the validation we show "loader" in the slug input indicating "pending" validation process. At the same time the save button is disabled.

How to test:
* `make start` to build and create test data
* login to RSD
* create software, try to use "add" as slug. You should get warning that "add" slug is already in use
* create project, tru

PR Checklist:
